### ISSUE TITLE
Improving invite by e-mail

### ISSUE THREAD

**Issue by brnsolikyl** [yunity/karrot-frontend#1984]
As it is now you can invite users to a group just by putting in the e-mail address and clicking the invite button. I think this can be improved, in the sense of making more personal invitations and also for the sake of group transparency.

- Personal invitations: meaning that users can send a customized invitation, writing their own text that would replace the standard text "Joe invited you to join "Does" community on karrot.world" on the body of the e-mail

- Transparency: meaning that the invitation and its content would be visible on the applications page and accessible on the members page. This is part of navigation improvement that I will propose and make more visual later.

**Comment by nicksellen** [comment]
I was actually wondering if we could _remove_ invitations entirely. The stats over at https://grafana.karrot.world tell me that in the last 30 days:

```
applications accepted/created = 105/123
invitations accepted/sent = 0/6
```

So, I don't think people are really using them. Also, they predated the application system, and I don't really feel it fits well with it either, the invitation mechanism will just add them directly to the group (so there is no "application" involved).

I would prefer to be able to slim down the featureset and make sure what we have is refined.

What do you think?

**Comment by brnsolikyl** [comment]
I didn't know it was so unused! Well, the button is kind of hidden...
It makes sense to remove it. I'm for it.

Btw, a related topic (maybe I should open another issue about it?) is to have invitations internal to Karrot, where user A invites user B who's in another group. Maybe not necessarily overriding the application process, but more like a little notification.

**Comment by tiltec** [comment]
A good replacement for email invitations would be to send the group link (e.g. https://karrot.world/#/groupPreview/37) to somebody, if the website would make it really straightforward to sign up and write an application.

**Comment by nicksellen** [comment]
With this approach it would be possible to remove the feature _and_ implement brunos ideas! (at least the custom message bit). The main bit to remove is that the software keeps track of the invitations and that they are more special than just visiting the group preview page.

**Comment by tiltec** [comment]
Maybe this issue can be closed and replaced by a new one, reflecting the last comments?

**Comment by tiltec** [comment]
Closed in favor of #2080
